gm4_poses/data/gm4_poses/functions/apply_book.mcfunction

```diff
@@ -0,0 +1,13 @@
+# @s = written book ..1 from armor_stand
+#copy player rotation to armor_stand part
+execute if entity @s[nbt={Item:{tag:{pages:["rotate"]}}},tag=!gm4_pose_changed] as @e[distance=..1,type=armor_stand,sort=nearest,tag=!gm4_no_edit] run data modify entity @s Rotation set from entity @p[distance=..1] Rotation
+execute if entity @s[nbt={Item:{tag:{pages:["head"]}}},tag=!gm4_pose_changed] run function gm4_poses:pose/head
+execute if entity @s[nbt={Item:{tag:{pages:["body"]}}},tag=!gm4_pose_changed] run function gm4_poses:pose/body
+execute if entity @s[nbt={Item:{tag:{pages:["arms"]}}},tag=!gm4_pose_changed] run function gm4_poses:pose/arms
+execute if entity @s[nbt={Item:{tag:{pages:["legs"]}}},tag=!gm4_pose_changed] run function gm4_poses:pose/legs
```

Comment:
The `tag` argument should go first so that the NBT isn't checked unless the `tag` matches